howsthewaterfront/src/styles/advanced-search.css

```diff
@@ -175,3 +175,36 @@
     width: 33%;
   }
 }
+
+.paginate-buttons {
+  margin-top: 5rem;
+  display: flex;
+}
+
+.previous {
+  margin-right: 13rem;
+  font-size: 2rem;
```

Comment:
Prev and Next button's font size should be 1.2 rem or 12px.

Reply:
Prev and Next buttons can be placed closer to each other. The spacing above and below the pagination buttons must be the same. Currently, the space between the buttons and footer is very little.